message_ix_models/model/material/data_generic.py

```diff
@@ -15,7 +15,7 @@
 from .util import read_config
 
 
-def read_data_generic(scenario):
+def read_data_generic(scenario) -> (pd.DataFrame, pd.DataFrame):
```

Comment:
```suggestion
def read_data_generic(scenario: Scenario) -> (pd.DataFrame, pd.DataFrame):
```
The more type hints we use, the better the feature becomes. This might require `from message_ix import make_df, Scenario`.